lib/services/ios-project-service.ts

```diff
@@ -1312,6 +1326,32 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		}
 		return teamId;
 	}
+
+	private async getEmbeddedPlistData(projectData: IProjectData): Promise<IDictionary<any>> {
+		let embeddedPlistContents = "";
+		const embeddedMobileProvisionPath = path.join(this.getPlatformData(projectData).deviceBuildOutputPath, `${projectData.projectName}.app`, "embedded.mobileprovision");
+
+		embeddedPlistContents = await this.$childProcess.exec(`security cms -D -i ${embeddedMobileProvisionPath}`);
+
+		return plist.parse(embeddedPlistContents);
+	}
+
+	private async getExportOptionsMethod(projectData: IProjectData): Promise<"app-store" | "ad-hoc" | "enterprise" | "development"> {
```

Comment:
Working with provisioning profiles.
This belongs to the https://github.com/NativeScript/nativescript-cli/blob/master/lib/services/ios-provision-service.ts#L9
Add a method there that by given path the mobileprovision can be read.